qiita_ware/test/test_processing_pipeline.py

```diff
@@ -41,8 +41,73 @@
 class ProcessingPipelineTests(TestCase):
     def setUp(self):
         self.db_dir = get_db_files_base_dir()
-        self.files_to_remove = []
-        self.dirs_to_remove = []
+
+        # Create a SFF dataset: add prep template and a RawData
+        study = Study(1)
+        md_dict = {
+            'SKB8.640193': {'center_name': 'ANL',
+                            'primer': 'GTGCCAGCMGCCGCGGTAA',
+                            'barcode': 'GTCCGCAAGTTA',
+                            'run_prefix': "preprocess_test",
+                            'platform': 'ILLUMINA',
+                            'library_construction_protocol': 'AAAA',
+                            'experiment_design_description': 'BBBB'},
+            'SKD8.640184': {'center_name': 'ANL',
+                            'primer': 'GTGCCAGCMGCCGCGGTAA',
+                            'barcode': 'CGTAGAGCTCTC',
+                            'run_prefix': "preprocess_test",
+                            'platform': 'ILLUMINA',
+                            'library_construction_protocol': 'AAAA',
+                            'experiment_design_description': 'BBBB'},
+            'SKB7.640196': {'center_name': 'ANL',
+                            'primer': 'GTGCCAGCMGCCGCGGTAA',
+                            'barcode': 'CCTCTGAGAGCT',
+                            'run_prefix': "preprocess_test",
+                            'platform': 'ILLUMINA',
+                            'library_construction_protocol': 'AAAA',
+                            'experiment_design_description': 'BBBB'}
+        }
+        md = pd.DataFrame.from_dict(md_dict, orient='index')
+        self.sff_prep_template = PrepTemplate.create(md, study, "16S")
+
+        tmp_dir = mkdtemp()
+        path_builder = partial(join, tmp_dir)
```

Comment:
A really minor thing would be to make this `path_builder` an attribute of the test so it can also be used in line 388.